### ISSUE TITLE
Micropython and CircuitPython Examples

### ISSUE THREAD

**Issue by darianbjohnson** [Phambili-Tech/Newt_Display#10]
Would love to make a mPy and cPy set of examples for users. The biggest hurdle is creating a library for the RTC (Microcrystal RV-3028).

**Comment by andypiper** [comment]
I think there's a MicroPython starter [here](https://github.com/CoreElectronics/CE-Makerverse-RV3028-MicroPython-Module), but it only covers a subset according to the README.

**Comment by darianbjohnson** [comment]
Yeah - I looked at this a while ago - there are a few functions that would need to be written - periodic timer, countdown timer, alarm, and frequency output (send to the screen to stop burn in). Still - I don't think it's "hard" to write - just takes time

**Comment by tjfoth** [comment]
Darian, have you looked into this?  I'm wondering if I should add it to my projects list of things I'd like to play with?

**Comment by darianbjohnson** [comment]
I haven't really made much progress on this... though it's still on my list... but you should feel free to start tinkering with ideas
